lib/formBuilder.tsx

```diff
@@ -136,46 +135,50 @@ function _buildForm(element: FormElement, lang: string): ReactElement {
       );
     case "checkbox": {
       const checkboxItems = choices.map((choice, index) => {
-        return (
-          <Checkbox
-            key={`${id}.${index}`}
-            id={`${id}.${index}`}
-            name={`${id}`}
-            label={choice}
-            required={isRequired}
-          />
-        );
+        return {
+          key: `${id}.${index}`,
+          id: `${id}.${index}`,
+          name: `${id}`,
+          label: choice,
+          required: isRequired,
+        };
       });
 
       return (
         <FormGroup name={id} ariaDescribedBy={description ? `desc-${id}` : undefined}>
           <div className="focus-group">
-            <legend className="gc-label">{labelText}</legend>
+            {labelComponent}
```

Comment:
This change is to make sure we show the asterisk and screen reader elements for a "required" field